groupease-server/src/main/java/io/github/groupease/group/GroupDao.java

```diff
@@ -0,0 +1,66 @@
+package io.github.groupease.group;
+
+import javax.annotation.Nonnull;
+import javax.inject.Inject;
+import javax.persistence.EntityManager;
+import javax.persistence.TypedQuery;
+
+import static java.util.Objects.requireNonNull;
+
+import io.github.groupease.model.Group;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.invoke.MethodHandles;
+import java.util.List;
+
+public class GroupDao {
```

Comment:
Missing JavaDocs.

Also, this doesn't use an interface like the channel package does.

Guice dependency injection and bindings make it really easy to code to interfaces, and really easy to swap out implementations later as needed (like I did with the `MockChannelDao` when I got Hibernate working).